the-blue-alliance-ios/View Controllers/Settings/SettingsTableViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  SettingsTableViewController.swift
+//  The Blue Alliance
+//
+//  Created by Anas Merbouh on 18-05-20.
+//  Copyright © 2018 The Blue Alliance. All rights reserved.
+//
+
+import UIKit
+import TBAKit
+
+class SettingsTableViewController: UITableViewController {
+    
+    // MARK: - View's lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    // MARK: - Table view configuration
+    
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        switch indexPath.section {
+        case 0:
+            var urlString: String?
+            
+            switch indexPath.row {
+            case 0:
+                urlString = "https://www.thebluealliance.com"
+                self.tableView.deselectRow(at: indexPath, animated: true)
```

Comment:
Move this deselect call to the very top of this method (like this) https://github.com/the-blue-alliance/the-blue-alliance-ios/blob/swift/the-blue-alliance-ios/View%20Controllers/Teams/TeamInfoTableViewController.swift#L164